src/components/ha-selector/ha-selector-text.ts

```diff
@@ -0,0 +1,38 @@
+import { customElement, html, LitElement, property } from "lit-element";
+import { fireEvent } from "../../common/dom/fire_event";
+import { HomeAssistant } from "../../types";
+import "@polymer/paper-input/paper-textarea";
+
+@customElement("ha-selector-text")
+export class HaTextSelector extends LitElement {
+  @property() public hass!: HomeAssistant;
+
+  @property() public value?: any;
+
+  @property() public label?: string;
+
+  protected render() {
+    return html`<paper-textarea
```

Comment:
I've updated the backend PR to make `text` selector have a `multiline` attribute. So we should only render a `textarea` when `multiline` is true.

Reply:
Fixed!